src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -162,6 +162,18 @@ public boolean hasPair(Pair pair) {
         return friendlyLink.hasPair(pair);
     }
 
+    @Override
+    public void addPair(Nric elderlyNric, Nric volunteerNric) {
+        friendlyLink.addPair(elderlyNric, volunteerNric);
+        // TODO: implement updateFilteredPersonList(PREDICATE_SHOW_ALL_PAIRS);
```

Comment:
Would it be too troublesome to add this to this PR? The function is already implemented, right?

Reply:
lets put this in another pull request to keep this PR small